tensorflow/python/compiler/tensorrt/trt_convert_test.py

```diff
@@ -508,7 +508,7 @@ def run(self, x):
     np_input = np.random.random_sample([5, 3]).astype(np.float32)
 
     def _InputFunc():
-      yield (np_input,)
```

Comment:
@tfeher: I've implemented all the changes you suggested except for the new unit test. 
I found that by a simple change of that line we will be able to test new functionality.

Reply:
@drivanov I am afraid that is not enough. We need more comprehensive test:
- have a input function that yields a single input (as the test above)
- input function that yields more than one input e.g.:
```python
yield input1
yield input2
```
- test network with 1 input and 2 inputs

Reply:
DONE:
- [a test](https://github.com/tensorflow/tensorflow/blob/bbb24f091f9650d580ff972cab0bf79771c4127a/tensorflow/python/compiler/tensorrt/trt_convert_test.py#L503-L505) with 6 subtests for one input;
- [a test](https://github.com/tensorflow/tensorflow/blob/bbb24f091f9650d580ff972cab0bf79771c4127a/tensorflow/python/compiler/tensorrt/trt_convert_test.py#L536-L538) with 5 subtests for two inputs.